mysql-test/suite/galera/r/galera_UK_conflict.result

```diff
@@ -69,3 +69,63 @@ f1	f2	f3
 INSERT INTO t1 VALUES (7,7,7);
 INSERT INTO t1 VALUES (8,8,8);
 DROP TABLE t1;
+test scenario 2
```

Comment:
In commit message please explain why it is false positive GAP lock conflict on secondary key.